app/utils/lib/RoomTypeConfig.js

```diff
@@ -214,6 +214,10 @@ export class RoomTypeConfig {
 		return false;
 	}
 
+	listenStreamerEvents() {
```

Comment:
I found this name confusing.. it is used to check if it will allow emit data.. I would change it to either `willEmitStreamerEvents` or `isEmitAllowed`